### DISCUSSION TITLE
Remove the "following / for you / more" bar?

### DISCUSSION THREAD

**Discussion by StarChaser** [AprilSylph/XKit-Rewritten#728] [Help Desk] [answered]
When you scroll up, a bar drops down with Following <3 For You (globe) More (down arrow) and it's very clunky and slows the site down. Do one of the modifications in Xkit kill this? I didn't see any obvious ones and am not sure what the bar is called to look more directly myself.  Couldn't find anything in a search here.

**Comment by marcustyphoon** [comment, marked as answer]
Yes! Well, sort of. The code that actually does the showing and hiding will still run, so the performance impact is still mostly there (especially on Firefox for weird reasons).

Anyway, I'm on my phone, but off the top of my head, the option name you're looking for is "remove the sticky effect from the dashboard tab bar," or something similar, in Tweaks.

**Reply by AprilSylph** [reply]
Note that this option is currently broken in the latest addon version, but will be fixed in the next release today or tomorrow.

**Comment by StarChaser** [comment]
Excellent, thanks.
